oleesir/store-manager
# #161182033Users should see a dashboard page

#### What does this PR do?
implement dashboard-template

#### Description of Task to be completed?
 implement dashboard-template

#### How should this be manually tested?
* Navigate to the UI folder and click on dashboard.html in your browser
* You will see a dashboard template


#### What are the relevant pivotal tracker stories?
[#161182033](https://www.pivotaltracker.com/story/show/161182033)

#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="screen shot 2018-10-12 at 2 29 08 pm" src="https://user-images.githubusercontent.com/17679511/46879168-327a4180-ce3d-11e8-8cc9-945040a61d3b.png">
